### PR TITLE
Replace ethereumjs/tx dependency with a fork that has ethers.js removed

### PR DESCRIPTION
**Pull request by avkos**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist for 1.x:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` with success.
- [ ] I have tested the built `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

## Checklist for 4.x:

- [ ] I have selected the correct 4.x base branch.
- [ ] Within the description, the feature or issue is discussed in detail or an issue is linked.   
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added any required tests for the changes I made 
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `yarn` successfully
- [ ] I ran `yarn lint` successfully
- [ ] I ran `yarn build:web` successfully
- [ ] I ran `yarn test:unit` successfully
- [ ] I ran `yarn test:integration` successfully
- [ ] I ran `compile:contracts` successfully
- [ ] I have tested my code.
- [ ] I have updated the corresponding `CHANGELOG.md` file in the packages I have edited.

### CODE REVIEW

File: packages/web3-utils/src/common/chains/rinkeby.json
Comment (Spacesai1or) [inline packages/web3-utils/src/common/chains/rinkeby.json:1]: This shouldn't be needed, as Rinkeby is [deprecated](https://blog.ethereum.org/2022/06/21/testnet-deprecation)

File: packages/web3-utils/src/common/chains/ropsten.json
Comment (Spacesai1or) [inline packages/web3-utils/src/common/chains/ropsten.json:1]: Ropsten is [deprecated](https://blog.ethereum.org/2022/06/21/testnet-deprecation)

File: packages/web3-utils/src/index.ts
Comment (mpetrunic) [inline packages/web3-utils/src/index.ts]: @web3/web3-js Might make sense to create `web3-eth-tx` package (or put in `web3-eth-accounts`) because in `web3-utils` it will get pulled in every other package and bloat package while in reality it's being used in `web3-eth-accounts` and `web3-eth` packages

File: packages/web3-utils/src/internal.ts
Comment (mpetrunic) [inline packages/web3-utils/src/internal.ts]:
```suggestion
	return str.startsWith('0x');
```
Don't you already have most of these methods in web3-utils and web3-validator?

File: packages/web3-utils/src/rlp/index.ts
Comment (mpetrunic) [inline packages/web3-utils/src/rlp/index.ts:1]: @ethereumjs/rlp has no dependencies so you can use it instead of pulling it in

File: packages/web3-utils/src/signature.ts
Comment (mpetrunic) [inline packages/web3-utils/src/signature.ts:1]: Pretty sure you have those methods in web3.js already (web3-eth-accounts)

File: packages/web3-utils/src/ssz.ts
Comment (mpetrunic) [inline packages/web3-utils/src/ssz.ts:1]: remove this file, I don't think you need it

File: packages/web3-utils/src/tx/depInterfaces.ts
Comment (mpetrunic) [inline packages/web3-utils/src/tx/depInterfaces.ts:1]: I don't think you need Kzg

File: packages/web3-utils/src/tx/kzg/kzg.ts
Comment (mpetrunic) [inline packages/web3-utils/src/tx/kzg/kzg.ts:1]: I don't think you need this

File: packages/web3-utils/src/tx/transactionFactory.ts
Comment (mpetrunic) [inline packages/web3-utils/src/tx/transactionFactory.ts]: remove this, it's not used in web3.js. You can also remove ethers dependency

File: packages/web3-utils/src/index.ts
Comment (mpetrunic) [inline packages/web3-utils/src/index.ts]: I don't think we need to add support for withdrawals just yet

File: packages/web3-utils/src/index.ts
Comment (mpetrunic) [inline packages/web3-utils/src/index.ts]: We don't need ssz just yet

File: packages/web3-utils/src/common/chains/mainnet.json
Comment (mpetrunic) [inline packages/web3-utils/src/common/chains/mainnet.json]: I think you can safely remove `bootstrapNodes` and `dnsNetworks` from every chain config as they are not used^^

File: packages/web3-utils/src/common/chains/goerli.json
Comment (jdevcs) [inline packages/web3-utils/src/common/chains/goerli.json]: We were blocking these files in past while bundling, if these are not used ( along with other functions ) we should not bring in our lib. https://github.com/web3/web3.js/blob/0781d8a95d0a1beae213fefaf234a6efed93b64e/webpack.base.config.js#L74

Comment (mpetrunic) [reply packages/web3-utils/src/common/chains/goerli.json]: @jdevcs you are suing those files but not genesisStates which they already removed from lib in latest version already^^

File: packages/web3-eth-accounts/package.json
Comment (mpetrunic) [inline packages/web3-eth-accounts/package.json]: You can remove this and all szz related types in types.ts as those are all related to danksharding eip which is not planned for mainnet anytime soon afaik

File: packages/web3-eth-accounts/src/rlp.ts
Comment (mpetrunic) [inline packages/web3-eth-accounts/src/rlp.ts:1]: Did you forget about rlp? 😄

File: packages/web3-eth-accounts/src/tx/utils/blobHelpers.ts
Comment (mpetrunic) [inline packages/web3-eth-accounts/src/tx/utils/blobHelpers.ts:1]: You don't need this file, it's related to danksharding eip 4844

**Comment by avkos** [comment]
refactor result
image

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (spacesailor24) [inline packages/web3-eth-accounts/src/common/utils.ts:1]: Wouldn't it make sense for these function to exist in `web3-utils`? I also think some of these methods could be replaced by existing utils

File: packages/web3-eth-accounts/src/rlp.ts
Comment (spacesailor24) [inline packages/web3-eth-accounts/src/rlp.ts:1]: Same with this, seems like it should belong in `web3-utils`

**Comment by spacesailor24** [comment]
I don't think it makes sense for all the files to be added to `web3-eth-accounts` package. Some files like:

- `packages/web3-eth-accounts/src/common/utils.ts`
    - I think some of the util functions in the files could be replaced (or could replace) with existing `web3-utils` functions
- `packages/web3-eth-accounts/src/rlp.ts`

should exist in `web3-utils`

- `packages/web3-eth-accounts/src/tx/legacyTransaction.ts`
- `packages/web3-eth-accounts/src/tx/baseTransaction.ts`
- `packages/web3-eth-accounts/src/tx/eip1559Transaction.ts`
- `packages/web3-eth-accounts/src/tx/eip2930Transaction.ts`

should exist in `web3-eth`, but that would cause a circular dependency issue with `web3-eth-accounts`

File: packages/web3-eth/test/integration/defaults.test.ts
Comment (Spacesai1or) [inline packages/web3-eth/test/integration/defaults.test.ts]: Pending `TODO`

File: packages/web3-providers-ipc/test/fixtures/helpers.ts
Comment (Spacesai1or) [inline packages/web3-providers-ipc/test/fixtures/helpers.ts]: Why was this change needed?

File: packages/web3-providers-ipc/test/unit/check_implementation.test.ts
Comment (Spacesai1or) [inline packages/web3-providers-ipc/test/unit/check_implementation.test.ts]: Why was this change needed?

File: packages/web3-utils/src/constants.ts
Comment (jdevcs) [inline packages/web3-utils/src/constants.ts]: This constant is used in eth-accounts package so it should be in eth-accounts instead of utils package.

File: packages/web3-utils/src/constants.ts
Comment (jdevcs) [inline packages/web3-utils/src/constants.ts]: same here:

> This constant is used in eth-accounts package so it should be in eth-accounts instead of utils package

.

File: packages/web3-utils/src/constants.ts
Comment (jdevcs) [inline packages/web3-utils/src/constants.ts]: MAX_INTEGER_BIGINT is not used any where? so it should be removed.

File: packages/web3-utils/src/constants.ts
Comment (jdevcs) [inline packages/web3-utils/src/constants.ts]: pls move all constants to package where these are used. Only keep constants in web3-utils if these are used by more than one packages.

File: packages/web3-utils/src/hash.ts
Comment (jdevcs) [inline packages/web3-utils/src/hash.ts]: pls avoid such changes and only white space changes in files , specially in large PRs.

File: packages/web3-validator/src/validation/string.ts
Comment (jdevcs) [inline packages/web3-validator/src/validation/string.ts:35]: We should also move some tests for these functions ( after this PR may be ) so we dnt drop coverage. With this PR its dropped 9% and we are below 80% goal. https://github.com/web3/web3.js/issues/5973

**Comment by jdevcs** [comment]
> I don't think it makes sense for all the files to be added to `web3-eth-accounts` package. Some files like:
> 
> * `packages/web3-eth-accounts/src/common/utils.ts`
>   
>   * I think some of the util functions in the files could be replaced (or could replace) with existing `web3-utils` functions
> * `packages/web3-eth-accounts/src/rlp.ts`
> 
> should exist in `web3-utils`
> 
> * `packages/web3-eth-accounts/src/tx/legacyTransaction.ts`
> * `packages/web3-eth-accounts/src/tx/baseTransaction.ts`
> * `packages/web3-eth-accounts/src/tx/eip1559Transaction.ts`
> * `packages/web3-eth-accounts/src/tx/eip2930Transaction.ts`
> * `packages/web3-eth-accounts/src/tx/transactionFactory.ts`
> * `packages/web3-eth-accounts/src/tx/types.ts`
> * `packages/web3-eth-accounts/src/tx/utils.ts`
> 
> should exist in `web3-eth`, but that would cause a circular dependency issue with `web3-eth-accounts`

If these functions are used in only accounts and eth package and we keep under same folder name like `tx` and `common`, 
- it will be easier to maintain for future maintainers ( if any update comes in ethereum-tx/common ) and if we want to apply in our lib.
- it will not blot other packages ( as utils is included in almost all other packages )

File: packages/web3-providers-ipc/test/fixtures/helpers.ts
Comment (avkos) [reply packages/web3-providers-ipc/test/fixtures/helpers.ts]: it didn't work when I test with ES2020. but then I return to our ES. so yes. I can revert these changes. thx

File: packages/web3-validator/src/validation/string.ts
Comment (avkos) [reply packages/web3-validator/src/validation/string.ts:35]: yes tests. is the next my step

File: packages/web3-providers-ipc/test/fixtures/helpers.ts
Comment (avkos) [reply packages/web3-providers-ipc/test/fixtures/helpers.ts]: fixed

File: packages/web3-providers-ipc/test/unit/check_implementation.test.ts
Comment (avkos) [reply packages/web3-providers-ipc/test/unit/check_implementation.test.ts]: fixed

File: packages/web3-utils/src/constants.ts
Comment (avkos) [reply packages/web3-utils/src/constants.ts]: done

File: packages/web3-utils/src/constants.ts
Comment (avkos) [reply packages/web3-utils/src/constants.ts]: done

File: packages/web3-utils/src/constants.ts
Comment (avkos) [reply packages/web3-utils/src/constants.ts]: done

File: packages/web3-utils/src/constants.ts
Comment (avkos) [reply packages/web3-utils/src/constants.ts]: done

File: packages/web3-utils/src/hash.ts
Comment (avkos) [reply packages/web3-utils/src/hash.ts]: done

**Comment by jdevcs** [comment]
With this PR:
image
Analyzed only main web3 package 
( https://statoscope.tech/ )

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/common/utils.ts:64]: Could we replace this with [numberToHex](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L177)?

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/common/utils.ts:329]: Could this replace our [bytesToBuffer](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L84)?

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/common/utils.ts:381]: We have [bytesToHex](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L127) method

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/common/utils.ts:389]: We could replace this with [toNumber](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L376) if we extend it to allow for Buffer

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/common/utils.ts:400]: Could replace with [toBigInt](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L410)

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/common/utils.ts:504]: [toHex](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L313)

File: packages/web3-eth-accounts/src/rlp/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/rlp/utils.ts]: [toHex](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L313)

File: packages/web3-eth-accounts/src/rlp/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/rlp/utils.ts]: [toHex](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L313)

File: packages/web3-eth-accounts/src/rlp/utils.ts
Comment (Spacesai1or) [inline packages/web3-eth-accounts/src/rlp/utils.ts]: [hexToBytes](https://github.com/web3/web3.js/blob/de5617ce4f0f50785e075c7638deeac1d6ed615d/packages/web3-utils/src/converters.ts#L140)

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (jdevcs) [reply packages/web3-eth-accounts/src/common/utils.ts:64]: +1 ( see if we can reuse existing functions in our lib )

**Review by Muhammad-Altabba — COMMENTED** [review]
Great job @avkos :+1: 

And because maintainability is sometimes more important than adding functionality. I suggest the following:

- Add all the code, that is imported from ethereumjs, inside a folder with no code other than the imported one.
- Add a readme file inside that folder that will keep track of the following:
  - The version (or the commit) of ethereumjs from which the code has been imported. We will need this for later to track changes and fix bugs.
  - What are the exact modifications done to every file, if any. (you can also do it this way for example: first push all the code taken from ethereumjs as-is without any modifications. So, we have a reference to track what exactly changed with further commits).
   - At every later update of the original code base (at ethereumjs) the corresponding folder could be updated and the readme file should give a detailed information about that update as well.
 - Import all the related tests from ethereumjs and place them under a specific folder that does not contain any other tests. (this could be done in another MR, I think).

**Comment by Spacesai1or** [comment]
> Great job @avkos +1
> 
> And because maintainability is sometimes more important than adding functionality. I suggest the following:
> 
> * Add all the code, that is imported from ethereumjs, inside a folder with no code other than the imported one.
> * Add a readme file inside that folder that will keep track of the following:
>   
>   * The version (or the commit) of ethereumjs from which the code has been imported. We will need this for later to track changes and fix bugs.
>   * What are the exact modifications done to every file, if any. (you can also do it this way for example: first push all the code taken from ethereumjs as-is without any modifications. So, we have a reference to track what exactly changed with further commits).
>   * At every later update of the original code base (at ethereumjs) the corresponding folder could be updated and the readme file should give a detailed information about that update as well.
> * Import all the related tests from ethereumjs and place them under a specific folder that does not contain any other tests. (this could be done in another MR, I think).

I think this is a great idea

**Comment by mpetrunic** [comment]
> > Great job @avkos +1
> > 
> > And because maintainability is sometimes more important than adding functionality. I suggest the following:
> > 
> > * Add all the code, that is imported from ethereumjs, inside a folder with no code other than the imported one.
> > * Add a readme file inside that folder that will keep track of the following:
> >   
> >   * The version (or the commit) of ethereumjs from which the code has been imported. We will need this for later to track changes and fix bugs.
> >   * What are the exact modifications done to every file, if any. (you can also do it this way for example: first push all the code taken from ethereumjs as-is without any modifications. So, we have a reference to track what exactly changed with further commits).
> >   * At every later update of the original code base (at ethereumjs) the corresponding folder could be updated and the readme file should give a detailed information about that update as well.
> > * Import all the related tests from ethereumjs and place them under a specific folder that does not contain any other tests. (this could be done in another MR, I think).
> 
> I think this is a great idea

I kinda disagree, you don't want most of the code and updates from ethereumjs. 
Its very trivial part of code, which we should maintain from now on by refactoring it and adding only new features(new tx types and chains) which we care about. 

Like, after release, we could refactor common and strip a lot of stuff.

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/common/utils.ts:64]: Our `numberToHex` can use negative or decimals numbers but `intToHex` - no

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/common/utils.ts:329]: toBuffer supports `Buffer`, `String` (hex-prefixed), `Number`, null/undefined, `BigInt` and other objects as an input.
our `bytesToBuffer` supports only `bytes`

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/common/utils.ts:381]: here is different input param type `byte` and `Buffer`

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/common/utils.ts:389]: `bufferToBigInt` use `intToHex` inside. so it can't allow to pass negative or decimals params

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/common/utils.ts:400]: it is not `TO` BigInt it is from BigInt

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/common/utils.ts:504]: if replace base test is not working. error:
``` Web3ValidatorError: Web3 validator found 1 error[s]:
    value "90062145607137483435911490614552348104830437995083346642150762831216482558573" at "/0" must pass "int" validation
```
so our `toHex` has extra validation

File: packages/web3-eth-accounts/src/rlp/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/rlp/utils.ts]: the same

File: packages/web3-eth-accounts/src/rlp/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/rlp/utils.ts]: the same

File: packages/web3-eth-accounts/src/rlp/utils.ts
Comment (avkos) [reply packages/web3-eth-accounts/src/rlp/utils.ts]: the same out hexToBytes has validation and fails by error:
`value "f889" at "/0" must pass "bytes" validation`

**Comment by jdevcs** [comment]
> > > Great job @avkos +1
> > > And because maintainability is sometimes more important than adding functionality. I suggest the following:
> > > 
> > > * Add all the code, that is imported from ethereumjs, inside a folder with no code other than the imported one.
> > > * Add a readme file inside that folder that will keep track of the following:
> > >   
> > >   * The version (or the commit) of ethereumjs from which the code has been imported. We will need this for later to track changes and fix bugs.
> > >   * What are the exact modifications done to every file, if any. (you can also do it this way for example: first push all the code taken from ethereumjs as-is without any modifications. So, we have a reference to track what exactly changed with further commits).
> > >   * At every later update of the original code base (at ethereumjs) the corresponding folder could be updated and the readme file should give a detailed information about that update as well.
> > > * Import all the related tests from ethereumjs and place them under a specific folder that does not contain any other tests. (this could be done in another MR, I think).
> > 
> > 
> > I think this is a great idea
> 
> I kinda disagree, you don't want most of the code and updates from ethereumjs. Its very trivial part of code, which we should maintain from now on by refactoring it and adding only new features(new tx types and chains) which we care about.
> 

Thanks everyone. We will maintain code added in this PR in future. Also if a readme file or comments are added for just tracking which version of eth/tx and eth/common lib were added it will be helpful for future maintainers to understand which imp bugs or upgrades are necessary in our lib by tracking eth/tx and eth/common ( just for info ).

> Like, after release, we could refactor common and strip a lot of stuff.

@avkos @spacesailor24 For replacing functions : As we discussed in last team call eth/tx / eth/common functions have different implementation as compared to ours in some cases. so I'll suggest to continue use functions from eth/tx eth/common **for now**, so we have stable transaction functionality, and in near future we can refactor for reusing our functions or strip unused pieces.

File: packages/web3-eth-accounts/src/common/utils.ts
Comment (jdevcs) [reply packages/web3-eth-accounts/src/common/utils.ts:64]: IMO, for now we can continue using functions from eth/tx eth/common lib for having stable tx functionality by merging this PR. and after that in near future we can optimize this part by reusing our lib functions and removing unused code.

**Comment by avkos** [comment]
@jdevcs  I've added versions here:
[TX](https://github.com/web3/web3.js/pull/5963/files#diff-df6fa5d853bf8010b8d3c5602cfbe17e0c3e08a8b203c3747766a53bd6238c7dR18)
[Common](https://github.com/web3/web3.js/pull/5963/files#diff-991b927c92e5d223faa5afd20eb5392413b25829b228cde0ff153421c040af22R17)
[RLP](https://github.com/web3/web3.js/pull/5963/files#diff-59171e6398ebfd006534c2983659cb44a3bb08bbeaef66009a0956f8228be84dR17)

File: packages/web3-eth-accounts/src/rlp/index.ts
Comment (jdevcs) [inline packages/web3-eth-accounts/src/rlp/index.ts]: These files should be removed and @ethereumjs/rlp
 lib dependency should be used back. Suggested by @mpetrunic

**Review by jdevcs — APPROVED** [review]

**Comment by jdevcs** [comment]
Before merging pls check all tests should be passing in CI.

File: packages/web3-eth-accounts/package.json
Comment (Muhammad-Altabba) [inline packages/web3-eth-accounts/package.json:60]: So, `@ethereumjs/rlp` is still used. Did you forget to push some code? Or, do I miss something?

The CHANGELOG.md contains:
> -   Moved @ethereumjs/tx, @ethereumjs/common, @ethereumjs/rlp code to our source code (#5963)

Thanks,

File: packages/web3-utils/package.json
Comment (Muhammad-Altabba) [inline packages/web3-utils/package.json]: What was the need to remove this, please? it was put to support tree shaking (https://webpack.js.org/guides/tree-shaking/) write?

File: yarn.lock
Comment (Muhammad-Altabba) [inline yarn.lock:370]: I am wondering why this is still included (the same for `ethereumjs/tx` below). Is it because `@ethereumjs/rlp` is still used?

**Review by Muhammad-Altabba — COMMENTED** [review]
Great job!
However, I have a few questions...

And a tiny point: I think this error text was missed updating:
https://github.com/web3/web3.js/blob/99ac66d9a7f905f1d43bb021dc965fddf2e29660/packages/web3-errors/src/errors/transaction_errors.ts#L290

File: yarn.lock
Comment (mpetrunic) [reply yarn.lock:370]: I think I checked that before and it's actually ganache and maybe harhat that pull those in^^

File: packages/web3-eth-accounts/package.json
Comment (avkos) [reply packages/web3-eth-accounts/package.json:60]: Removed from changelog, thanks.
@jdevcs asked me to revert changes for RLP [here](https://github.com/web3/web3.js/pull/5963#discussion_r1159627500)

File: packages/web3-utils/package.json
Comment (avkos) [reply packages/web3-utils/package.json]: reverted

**Review by Muhammad-Altabba — APPROVED** [review]

**Pull request merged**
